docs/scripts/add_version.py

```diff
@@ -0,0 +1,94 @@
+"""Utilities for adding versions when building the documentation."""
+
+import sys
+from pathlib import Path
+
+
+def add_version_to_selector_page(version: str) -> None:
```

Comment:
The heading on the version selector page is missing a plural-"s", i.e. "Available version*s*"